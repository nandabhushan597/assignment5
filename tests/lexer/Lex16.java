'"'
  '\"'
  '\''
  '\\'
  'b'
  '\60'
  '\n'
  
