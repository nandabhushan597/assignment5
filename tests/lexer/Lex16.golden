Right [LiteralT (CharL 34)
LiteralT (CharL 34)
LiteralT (CharL 39)
LiteralT (CharL 92)
LiteralT (CharL 98)
LiteralT (CharL 48)
LiteralT (CharL 10)]